ember-app/app/mixins/cards/card-move.js

```diff
@@ -20,17 +20,19 @@ var CardMoveMixin = Ember.Mixin.create({
       var below_order = issue_below._data[this.get("orderKey")];
       return (above_order + below_order) / 2;
     },
-    moveToTop: function(issue_below){
-      var below_order = issue_below._data[this.get("orderKey")];
+    moveToTop: function(issue_below, key){
+      key = key ? key : this.get("orderKey");
```

Comment:
`key || this.get("orderKey")`

That said, can we just kill the `orderKey` property altogether? This implict statefulness seems a likely source of other bugs.